docs/LanguageGuide.md

```diff
@@ -217,6 +219,23 @@ Sometimes, it's more convenient to bring a module's declarations into scope only
 (Vector2.update-x my-pos inc) ;; => (Vector2 11 20)
 ```
 
+### Enums / Algebraic types
+```clojure
+(deftype (Either a b)
+  (Left a)
+  (Right b))
+
+;; A Variant can be created with the same syntax as call expression
+(Left 10)
```

Comment:
You either need to `use` `Either` before you can do that, or you’ll have to write `Either.Left`.